segmentstore/storage/src/test/java/io/pravega/segmentstore/storage/chunklayer/ChunkedSegmentStorageMockTests.java

```diff
@@ -326,13 +326,16 @@ public void testIOExceptionDuringWrite() throws Exception {
     public void testFileNotFoundExceptionDuringGarbageCollection() throws Exception {
         String testSegmentName = "test";
         SegmentRollingPolicy policy = new SegmentRollingPolicy(2); // Force rollover after every 2 byte.
-        val config = ChunkedSegmentStorageConfig.DEFAULT_CONFIG.toBuilder().defaultRollingPolicy(policy).build();
+        val config = ChunkedSegmentStorageConfig.DEFAULT_CONFIG.toBuilder()
+                .garbageCollectionFrequencyInSeconds(0)
+                .build();
 
         BaseMetadataStore spyMetadataStore = spy(new InMemoryMetadataStore(executorService()));
```

Comment:
Add `@Cleanup` to all the objects here that implement `AutoCloseable`

Reply:
done